std/uni.d

```diff
@@ -1760,17 +1771,34 @@ alias sharSwitchLowerBound = sharMethod!switchUniformLowerBound;
     static void append(T, V)(ref T[] arr, V value)
         if (!isInputRange!V)
     {
+        if (arr.length == size_t.max) assert(0);
```

Comment:
That's the uncovered line pointed out by CodeCov. Going back to a recent discussion, maybe it could be written like this and thus also be more helpful to the user?

```d
assert(arr.length < size_t.max, "maximal array size reached")
``

... or even better as input contract ;-)